PluginCore.Events.cs

```diff
@@ -17,6 +17,7 @@ private void CharacterFilter_LoginComplete(object sender, EventArgs e)
                 Core.CharacterFilter.Death += CharacterFilter_Death;
 
                 TriggerWebhooksForEvent(EVENTS.LOGIN, Core.CharacterFilter.Name + " has logged in.");
+                ChatPatterns.Add(new ChatPattern(EVENTS.RARE, Core.CharacterFilter.Name + " has discovered "));
```

Comment:
The rest of the pre-defined chat patterns are defined over in

https://github.com/amoeba/TownCrier/blob/a279c21d982b5c57e87680988dc453890a4243ca/PluginCore.cs#L72-L76

is there a reason you wanted to add that here in the `LoginComplete` trigger? I think I'd rather have them in one place.

Reply:
Yeah, I tried putting it there first and beat my head against it for a bit.  I don't think the charfilter.name works until login is complete. When I tried it in PluginCore.cs it was catching anything matching " has discovered " and wasn't limiting captures to the name.

Reply:
You know, I think you're right! I'll merge this as-is.